oaknational/Oak-Web-Application
# [CUR-826] Added lot picker to pathways

## Description

Music year: 1950

- Replaced `examboard` in `subjectPhaseOptions` with `ks4_options` to also include pathway
- Added sorted ks4Options into lot picker
- Updated tests and fixtures to use `ks4Options` instead of `examboard` (only where relates to lot picker, did not do this for visualiser / `curriculumUnits`
- Updated tests to remove `All examboards` in the CurriculumHeader as this option is no longer available
- Styling has not been updated yet as waiting for lot picker changes to be merged 

## How to test

1. Go to https://deploy-preview-2805--oak-web-application.netlify.thenational.academy
2. For running locally, ensure that the constants.ts file has Cycle 2 subjects enabled:
  ```export const ENABLE_CYCLE_2 = true;```
3. Click on lot picker - PE, Computing, Citizenship, RE
4. You should see Core +/ GCSE options
5. RHSE should have no options as this isn't available
6. Computing should have [Examboard] (Computer Science)

## Screenshots

How it should now look:
<img width="643" alt="Screenshot 2024-09-19 at 18 43 45" src="https://github.com/user-attachments/assets/45e0c0f7-5564-40aa-adb5-bf7377cce2e3">

## Checklist
